src/test/kotlin/interactor/GetCitiesHasTheCheapestBananaPricesTest.kt

```diff
@@ -0,0 +1,123 @@
+package interactor
+
+import io.mockk.*
+import io.mockk.impl.annotations.MockK
+import model.*
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertThrows
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.function.Executable
+
+class GetCitiesHasTheCheapestBananaPricesTest{
+
+    @MockK
+    private lateinit var cheapestBanana: GetCitiesHasTheCheapestBananaPricesInteractor
+    @MockK
+    lateinit var dataSource: CostOfLivingDataSource
+    @MockK
+    private lateinit var oneCityWithValidBananaPrice: CityEntity
+    @MockK
+    private lateinit var cityWithZeroPrice: CityEntity
+    @MockK
+    private lateinit var cityWithNegativePrice: CityEntity
+    @MockK
+    private lateinit var theAveragePriceCityGaza : CityEntity
+    @MockK
+    private lateinit var theMostCheapestCityBaghdad: CityEntity
```

Comment:
grammer mistake in namming your variable. make sure to refactor it (use shift+f6).
```suggestion
    private lateinit var theCheapestCityBaghdad: CityEntity
```

Reply:
done